apps/sps-ordbokene-web/components/ArticleHeader.vue

```diff
@@ -64,6 +56,7 @@ h3 {
 
 .hgno {
```

Comment:
What about if we use rounded-full with bg-tertiary? Because seems like the current background isn't effective.
<img width="269" alt="issue" src="https://github.com/uib-ub/uib-ub-monorepo/assets/43489196/3722f48e-24de-4f24-be98-3fab72f2c775">
<img width="111" alt="issue" src="https://github.com/uib-ub/uib-ub-monorepo/assets/43489196/c5a2905a-3a58-430c-9f69-874fc063a7e6">

Reply:
I'll remove the backround alltogether, it isn't necessary